api/lib/router/routes/admin/admin-scanner.ts

```diff
@@ -93,26 +113,26 @@ export class AdminScannerController extends ScannerController implements IExpres
   /**
    * @api {post} /admin/scanner/assign Assign RFID tags ID to users
    * @apiVersion 2.0.0
-   * @apiName Assign an RFID to a user (Admin)
+   * @apiName Assign an RFID to a user
    *
    * @apiGroup Admin Scanner
    * @apiPermission TeamMemberPermission
-   *
+   * @apiPermission ScannerPermission
+   * @apiUse ApiKeyArgumentRequired
    * @apiUse AuthArgumentRequired
    * @apiParam {Array} assignments An array or single instance of RFID tags to User uid assignments
    * @apiParamExample {json} Request-Example:
    *     [
```

Comment:
Update param example to have full request. (missing assignments: {...})

Reply:
Done